exercises/sieve/src/main/java/Sieve.java

```diff
@@ -0,0 +1,11 @@
+import java.util.List;
+
+public class Sieve {
+	public Sieve(int maxPrime) {
+		throw new UnsupportedOperationException("Delete this statement and provide your own implementation.");
+    }
+
+    public List<Integer> getPrimes() {
+    	throw new UnsupportedOperationException("Delete this statement and provide your own implementation.");
+    }
+}
```

Comment:
Minor nitpick, but if you're using IntelliJ there's an option to have it automatically add trailing newlines to files on save to avoid these warnings.